addons/portal/views/portal_templates.xml

```diff
@@ -15,8 +15,7 @@
                         <span class="navbar-toggler-icon"/>
                     </button>
                     <div class="collapse navbar-collapse" id="top_menu_collapse">
-                        <!-- FIXME We want menu to open to the right by default (except cases handled in JS) -->
-                        <ul class="nav navbar-nav ml-auto text-right" id="top_menu">
+                        <ul class="nav navbar-nav ml-auto text-right-not-sm" id="top_menu">
```

Comment:
did you mean ? text-md-right ?

But since the comment of @qsm-odoo "(except cases handled in JS)"
Their are probably other use case for some themes and co...

Maybe we could wait his pov ?

Reply:
thanks for the hint with `text-md-right` that is indeed what i should have used.